### DISCUSSION TITLE
Chips and icon list?

### DISCUSSION THREAD

**Discussion by hkr086** [Blackymas/NSPanel_HA_Blueprint#574] [General]
Hi, were do I find the chips and icon list I can use please?

**Comment by lloydbayley** [comment]
Icons can be found [here](https://htmlpreview.github.io/?https://github.com/jobr99/Generate-HASP-Fonts/blob/master/cheatsheet.html). Click on the U symbol next to the one you want (which copies the proper code) and paste it into the icon field for the entity/switch you want to use it for. When you paste it, it might show a funny character. This is normal and it will look fine on the display.

**Reply by hkr086** [reply]
Thank you!

**Comment by deejaybeam** [comment]
a link to this page is also in the blueprint itself:
![image](https://user-images.githubusercontent.com/10434729/229248120-cae45f73-aa4b-4da1-b19e-3dbc678703a9.png)
... and of course in the wiki

**Reply by hkr086** [reply]
Thank you guys!!
